hw3/scripts/Homework0825.txt

```diff
@@ -0,0 +1,135 @@
+#PART 1: Test the hypothesis that the average activation difference across subjects for Words > fixation (the Words contrast at the single subject level) is greater than zero.
+
+#Mask out non-brain regions and coverage differences
+across subjects
+3dmask_tool -input export/masks/full_mask.sub-* -frac 1.0 -prefix mask
+
+#Choose the correct sub-brick
+#use 3dinfo to explore the options (pick a random participant)
+3dinfo -label export/betas/stats.sub-08_REML+tlrc.BRIK
+
+#Look up index of Words#0_Coef - this didn't work :-()
+#3dinfo -label2index ’Words#0_Coef’ export/betas/stats.sub-08_REML+tlrc.BRIK
+
+#run a t test
+3dttest++ -mask mask+tlrc -prefix words-baseline -setA 'export/betas/stats*+tlrc.HEAD[Words#0_Coef]'
+
+#Copy or link an anatomical file for viewing 
+3dcopy ~/abin/MNI_avg152T1+tlrc .
+
+#PART 2: Cluster correction
+
+# create an average blur file; extract the relevant row
+
+tcsh:
+foreach f (export/blurs/blur*.1D)
+	1d_tool.py -infile $f -select_rows 3 -write - >> blurs.acf.1D
```

Comment:
This selects the row corresponding to OLS results, but I gave you REML results. The effect should be minor here, but note that the estimates are different depending on how the models are estimated.